documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -215,6 +279,38 @@ For the Java version of samm-cli, the full command would result in:
 java -jar samm-cli-{esmf-sdk-version}.jar aspect _AspectModel.ttl_ to openapi -b "https://www.example.org" -r "/resources/\{resourceId}" -p _fileLocation_
 ----
 
+=== Configuration of security schemas
+
+For example, to configure OAuth2 authentication the template could look like this:
+
+[source,yaml]
+----
+security:
+  - OAuth2:
+      - read:aspects
+
+components:
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      flows:
+        implicit:
+          authorizationUrl: https://{environment}.digitaltwin.eclipse.org/oauth2/authorize
+          tokenUrl: https://{environment}.digitaltwin.eclipse.org/oauth2/token
```

Comment:
Can we use something that does not look "official"? We should probably base this on "example.com", because there is no authentication provider at digitaltwin.eclipse.org.